pasteit/public/javascripts/pasted.js

```diff
@@ -5,6 +5,7 @@ $(document).ready(function() {
         mode = $('#mime-type').val();
     else
         mode = $("#selected-mode").val();
+    var humanreadable = getLangKey(mode);
```

Comment:
Use consistent variable naming. This should be humanReadable, not humanreadable.

Reply:
done in 25e11d9